analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -581,6 +581,15 @@ class Pub(
         return yamlMapper.readTree(definitionFile)
     }
 
+    override fun getVersion(workingDir: File?): String {
+        val result = ProcessCapture(workingDir, command(workingDir), getVersionArguments())
+        if (result.isError) {
+            throw IOException(result.errorMessage)
+        }
```

Comment:
But `run` is implemented as

    ProcessCapture(workingDir, command(workingDir), *args).requireSuccess()

which does not look much different than your code. What am I missing?

Reply:
Oops, sorry, I missed that `Pub` also overrides `run()`. But could we use `.requireSuccess()` instead of the custom `if`-clause?